com.unity.shadergraph/Editor/Generation/Targets/BuiltIn/Editor/ShaderGraph/Includes/LightingMetaPass.hlsl

```diff
@@ -1,10 +1,139 @@
 #ifndef SG_LIT_META_INCLUDED
 #define SG_LIT_META_INCLUDED
 
+#include "UnityMetaPass.cginc"
+#include "Packages/com.unity.shadergraph/Editor/Generation/Targets/BuiltIn/ShaderLibrary/SurfaceData.hlsl"
+
+SurfaceData SurfaceDescriptionToSurfaceData(SurfaceDescription surfaceDescription)
```

Comment:
These were normally shared functions for all built-in passes, but due to the surface description not containing certain fields (like metallic) in this pass I had to copy them out and remove the references to fields not in this pass.